setup.py

```diff
@@ -0,0 +1,19 @@
+import distutils
+from distutils.core import setup
+import glob
+
+bin_files = glob.glob("bin/*")
+data_files = [("",["setpath.sh"])]
+
+# The main call
+setup(name='salpytools',
+      version ='OCS_IntSpecICC',
```

Comment:
Is this really tracking the XML version? I didn't think there was much that was XML specific as this is an abstraction package. How does the version change if changes are only code but not the XML?

Reply:
Is this why the header service files are showing me an odd `HEADVER`? I was expecting some kind of number or SHA1 to appear for header version and it should be related to the header service configuration, not the code.

Reply:
This is a leftover from a previous integration activity. Unfortunately we do not have a method/definition to capture versions.